flowclient/flowclient/client.py

```diff
@@ -455,6 +455,320 @@ def daily_location(
     }
 
 
+def _meaningful_locations(
+    start_date: str,
+    stop_date: str,
+    label: str,
+    labels: Dict[str, Dict[str, dict]],
+    tower_cluster_radius: float = 1.0,
+    tower_cluster_buffer: float = 0.0,
+    tower_cluster_call_threshold: int = 0,
+    tower_dow_scores: Dict[str, float] = {
+        "monday": 1,
+        "tuesday": 1,
+        "wednesday": 1,
+        "thursday": 0,
+        "friday": -1,
+        "saturday": -1,
+        "sunday": -1,
+    },
+    tower_hour_of_day_scores: List[float] = [
```

Comment:
Similar to `tower_dow_scores`, not sure if we can provide a generically useful default value here.